cert/spec.go

```diff
@@ -529,37 +513,21 @@ func (spec *Spec) renewPKI() error {
 		}
 
 		spec.ResetBackoff()
+		spec.expiry.Cert = spec.Certificate().NotAfter
+		if spec.CA.File != nil {
+			err = spec.CA.File.WriteFile(pem.EncodeToMemory(&pem.Block{
+				Type:  "CERTIFICATE",
```

Comment:
nit/followup PR: Since we're doing this PEM encoding in a couple of different places in this commit (unsure if in other places in the code also), it should maybe be factored out into another method.